src/themes/OLH/templates/journal/article.html

```diff
@@ -119,9 +119,9 @@ <h3>{{ article.title|safe }}</h3>
                     <h3>{{ article.title|safe }}</h3>
                 <hr/>
                 {% endif %}
-                            {% if article.abstract != '' %}
+                            {% if article.abstract and article.abstract != ''%}
```

Comment:
Doesn't `if article.abstract` already account for falsey values?

Reply:
I think we've seen wierdness with this in the past so I've left if in. Can remove though if you think it'll be ok.